weasyl/moderation.py

```diff
@@ -364,11 +364,10 @@ def finduser(targetid, username, email, dateafter, datebefore, excludesuspended,
     if targetid:
         q = q.where(lo.c.userid == targetid)
     elif username:
-        q = q.where(lo.c.login_name.op('~')(username))
+        q = q.where(lo.c.login_name.op('ilike')('%%%s%%' % username.replace('\\', r'\\')))
```

Comment:
Are we on a recent enough python to use f-strings e.g. `f"%{username_r}%"` here, and break out the `username.replace` into a string assignment above?

Reply:
Yeah, f-strings are 3.6, and the Dockerfile is for 3.10